index.html

```diff
@@ -273,16 +273,42 @@ <h2 class="title-block__name _title">Work experience</h2>
                     </div>
                   </div>
                   <span class="work-card__post _title-font"
-                    >Junior developer</span
+                    >Senior developer</span
                   >
                 </div>
                 <div class="work-card__main">
+                  <h2 class="work-card__category">Info</h2>
                   <p class="work-card__text">
-                    The company is developing a second version of the product. I
-                    am supporting 2 projects: SPA UI and Business logic/API for
-                    working with data <br />Responsibilities: Interface
-                    development, adding new functionality, supporting legacy
-                    functionality, writing high-load database queries.
+                    The team is develops online surveys with complex
+                    functionality. Project contains SPA UI and Business
+                    logic/API for working with data.
+                  </p>
+                  <h2 class="work-card__category">Responsibilities</h2>
+                  <p class="work-card__text">
```

Comment:
Don't use `<ul>` in `<p>`, because `<p>` must contain only text, not list.

Reply:
Sorry, I mean we must delete `<p>` and use only `<ul>` + `<li>` (return `<ul>` and delete `<p>`)